application/language/zh_cn/project_launch_template_lang.php

```diff
@@ -44,7 +44,7 @@
 $lang['project_launch_template_end_silence'] = '在录音结尾留出5秒空白。';
 $lang['project_launch_template_filename'] = '文件名：';
 $lang['project_launch_template_check_back'] = '大约一天后[b]回本页看看[/b]是否有人提出了修改意见。';
-$lang['project_launch_template_deadline_poetry'] = '[b]截止时间：[/b]请在日期周日 GMT（格林威治时间） 06:00（即北美中部标准时间 12:00 AM）前提交你的录音。';
+$lang['project_launch_template_deadline_poetry'] = '[b]截止时间：[/b]请 %s GMT（格林威治时间） 06:00（即北美中部标准时间 12:00 AM）前提交你的录音。';
```

Comment:
The other languages I can guess that the change looks correct, but here I have no idea. I'm assuming you did your homework and that this is fine :)

Reply:
@twinkietoes-on did ask some Chinese speakers on the LibriVox forum for some assistance on this one.  In some ways it doesn't matter much as the poetry projects are currently only being used for English anyway but I still wanted to get reasonable results for the other languages.